src/site/content/en/blog/performance_tool_score_diffs.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Performance Tooling Score Differences
+subhead: I get different scores in Test My Site, PageSpee Insights, and Lighthouse - why?
+authors:
+  - egsweeny
+date: 2019-07-05
+hero: hero.jpg
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+alt: A description of the hero image for screen reader users.
+description: |
+  This post serves to explain the reasons that there are differences in performance scores between Test My Site (TMS), PageSpeed Insights (PSI), and Lighthouse. This document does not detail other differences in the tools, such as server run locations, network throttling presets, etc.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+# Why is my performance score different between different tools? What is my REAL score?
+
+<a href="https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics">Load is not a single moment in time</a>—it's an experience that no one metric can fully capture. There are multiple moments during the load experience that can affect whether a user perceives it as "fast", and if you just focus on one you might miss bad experiences that happen during the rest of the time. <b>This is why we generally use multiple metrics blended together to create our performance scores;</b> but different tools use different metrics and different weighted blends. This can be frustrating when you're looking for a source of truth about how fast your website is.
```

Comment:
```suggestion
<a href="https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics">Load is not a single moment in time</a>—it's an experience that no one metric can fully capture. There are multiple moments during the load experience that can affect whether a user perceives it as "fast", and if you just focus on one you might miss bad experiences that happen during the rest of the time. **This is why we generally use multiple metrics blended together to create our performance scores;** but different tools use different metrics and different weighted blends. This can be frustrating when you're looking for a source of truth about how fast your website is.
```